screens/Scan/ScanLayoutController.ts

```diff
@@ -63,6 +63,14 @@ export function useScanLayout() {
   const onStayInProgress = () =>
     scanService.send(ScanEvents.STAY_IN_PROGRESS());
   const onRetry = () => scanService.send(ScanEvents.RETRY());
+  const GOTO_HOME = () => {
```

Comment:
Can we also merge this to something like
```
const GOTO = (screen) => {
    scanService.send(ScanEvents.DISCONNECT());
    navigation.navigate(screen);
  };
```

Reply:
let's keep it separate for now